api/server/models/email.js

```diff
@@ -0,0 +1,46 @@
+var mandrill = require('mandrill-api/mandrill');
+var mandrill_client = new mandrill.Mandrill('Ca4nS3QStEcpvZdk9iMh0Q');
+
+module.exports = function (Email) {
+    Email.send_email = function (email_type, email_data, callback) {
+
+        var e_to = email_data.email;
+
+        email_html = Email.get_email_html(email_type, email_data);
+
+        e_body = email_html.body;
+        e_subject = email_html.subject;
+
+        Email.send({
+            to: e_to,
+            from: 'manish@excellencetechnologies.in',
+            subject: e_subject,
+            //text: 'Test Chatt',
+            html: e_body
+        }, function (err, mail) {
+            console.log('email sent!');
+            callback();
+        });
+    };
+    Email.get_email_html = function (type, data) {
```

Comment:
can you create different functions for different types of email instead of if conditions